dtsync/option.go

```diff
@@ -23,6 +24,14 @@ func (c *config) apply(opts []Option) error {
 	return nil
 }
 
+// MinerID sets a miner ID to include in the pubsub message.
+func MinerID(minerID string) Option {
```

Comment:
this can be general at this interface too: `WithExtraData(data []byte)`

Reply:
I too think we need to keep this Filecoin agnostic.